routes/taskRequests.js

```diff
@@ -12,4 +12,6 @@ router.get("/:id", authenticate, authorizeRoles([SUPERUSER]), taskRequests.fetch
 router.post("/addOrUpdate", authenticate, validateUser, taskRequests.addOrUpdate);
 router.patch("/approve", authenticate, authorizeRoles([SUPERUSER]), validateUser, taskRequests.approveTaskRequest);
 router.post("/", authenticate, validators.postTaskRequests, taskRequests.addTaskRequests);
+router.post("/migrations", authenticate, authorizeRoles([SUPERUSER]), taskRequests.migrateTaskRequests);
```

Comment:
`migrations` isn't a noun. can you please update this.

Reply:
- `migrate` is a verb and `migrations` is a noun. 
- Other names I can think about are `/scripts` or `/migration-scripts`. Let me know what you suggest
Also note these are a one time script